### ISSUE TITLE
Run pipeline is failed

### ISSUE THREAD

**Issue by nguyen-tien-mulodo** [xgeek-net/pipeline#17]
Hi Admin,

After I setup 2 connections and new pipeline successfully, run pipeline is always failed with error below:

14:14:40 [SF.api] Authorize : true
14:14:40 [Git] Clone from [git repo.git] (#pipeline ):
14:17:47 [Git] Pull files from branch: #pipeline
14:17:47 [ERROR] Run pipeline : Error: ENOENT: no such file or directory, stat '/Users/mypc/Library/Application Support/pipeline/pipeline/481289da-6fec-49f9-99b4-5a136891af66/git/tabs/ONG_Education__c.tab'

Could you please check and help me? Thank you.

**Comment by xgeek-net** [comment]
Hi, @nguyen-tien-mulodo , Which git service are you using? Does it work when you remove tabs metadata?

**Comment by nguyen-tien-mulodo** [comment]
@xgeek-net I understand error above, but now I have new error: 

[ERROR] Run pipeline : Error: EEXIST: file already exists,

Could you please help me? Thank you.

**Comment by xgeek-net** [comment]
@nguyen-tien-mulodo , seems the `src` folder is not found in root directory of git. Assign the src parent path when you create a pipeline as below:

![スクリーンショット 2019 06 19 19 44 36](https://user-images.githubusercontent.com/5466487/59759323-c81e7880-92ca-11e9-82a3-466a1cd5c6d6.png)

In case of the `src` folder is committed as path `parentfolder/src`.

**Comment by nguyen-tien-mulodo** [comment]
@xgeek-net the error is still occurred when I changed parent folder as you said above. 
Could you please give me sample git for this deployment? Thank you.

**Comment by xgeek-net** [comment]
@nguyen-tien-mulodo folk this repository, set root path to be blank, and try again.
https://github.com/xgeek-net/sfdc-apex-trigger-framework

**Comment by nguyen-tien-mulodo** [comment]
Hi @xgeek-net 

I run your repo above, but the deployment is still failed.
![21](https://user-images.githubusercontent.com/13327487/59817696-2854ff00-934b-11e9-925b-b707f7010c4b.png)

Please check attachment image. Thank you.

**Comment by xgeek-net** [comment]
@nguyen-tien-mulodo , it works for me, see below: 

<img width="1230" alt="スクリーンショット 2019-06-20 14 51 30" src="https://user-images.githubusercontent.com/5466487/59824054-0a979200-936b-11e9-830a-f2cde40058f3.png">

```
14:50:37 [SF.api] Authorize : true
14:50:37 [Github] Pull from sfdc-apex-trigger-framework (#master ):
14:50:37 [Github] Pull files from branch: #master
14:50:37         > .gitignore (Ignored)
14:50:37         > LICENSE (Ignored)
14:50:37         > README.md (Ignored)
14:50:39         > src/classes/TriggerHandler.cls-meta.xml (Size: 174)
14:50:39         > src/classes/AccountTriggerHandler.cls (Size: 1609)
14:50:39         > src/classes/TriggerHandler.cls (Size: 3031)
14:50:39         > src/triggers/AccountTrigger.trigger-meta.xml (Size: 178)
14:50:39         > src/classes/TriggerHandlerManager.cls-meta.xml (Size: 174)
14:50:39         > src/classes/AccountTriggerHandler.cls-meta.xml (Size: 174)
14:50:39         > src/triggers/AccountTrigger.trigger (Size: 588)
14:50:39         > src/classes/TriggerHandlerManager.cls (Size: 3607)
14:50:39 [Github] Pull Done.
```

**Comment by nguyen-tien-mulodo** [comment]
@xgeek-net I tried many times but they are failed :( 

![Screen Shot 2019-06-20 at 2 30 03 PM](https://user-images.githubusercontent.com/13327487/59833413-2b161b00-9370-11e9-9f42-b7859b9dfe4a.png)

Step by step below:

- Fork your sample repo to my own repo.
- Create connections: git and sandbox environment.
- Create new pipeline between git vs sandbox with name pipeline and branch to deploy (master). Leave all fields as their default values on this tool.
- Click Run pipeline and get result is failed. :(

Could you please help me? Thank you.

**Comment by xgeek-net** [comment]
@nguyen-tien-mulodo  could you share Metadata.zip and deploy log? you can get them as below : 

![59824054 0a979200 936b 11e9 830a f2cde40058f3](https://user-images.githubusercontent.com/5466487/59834661-4dfdfa80-9383-11e9-8bc7-f507897b9975.png)

**Comment by nguyen-tien-mulodo** [comment]
[de5938f6-39b2-498e-ab42-61e809e0de04.zip](https://github.com/xgeek-net/pipeline/files/3309602/de5938f6-39b2-498e-ab42-61e809e0de04.zip)
please download and check @xgeek-net . Thank you.

**Comment by xgeek-net** [comment]
@nguyen-tien-mulodo reproduced! Seems is a bug and I will fix it ASAP!

**Comment by nguyen-tien-mulodo** [comment]
Thank you so much @xgeek-net x
I'm waiting for your new version <3

**Comment by xgeek-net** [comment]
@nguyen-tien-mulodo , just fixed this issue at latest version [(2.0.2)](https://github.com/xgeek-net/pipeline/releases/tag/2.0.2), get update and try again. :)

**Comment by nguyen-tien-mulodo** [comment]
@xgeek-net thank you so much, not it's working well on my PC. 
Nice weekend @xgeek-net
